lisa/tools/kdump.py

```diff
@@ -0,0 +1,326 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PurePosixPath
+from typing import Dict, List, Type, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Debian, OperatingSystem, Posix, Redhat, Suse
+from lisa.tools import Cat, Sed
+from lisa.tools.service import Service
+from lisa.util import LisaException
+
+
+class KernelDumpTool:
```

Comment:
This class makes the dependencies and logic complex. It doesn't need such a tool to wrap them together. For each tool, it can check which command name should be in its logic.

Reply:
Done